dotnet/samples/GettingStartedWithTextSearch/Step4_Search_With_VectorStore.cs

```diff
@@ -0,0 +1,176 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Azure.Identity;
+using Microsoft.SemanticKernel.Connectors.AzureOpenAI;
+using Microsoft.SemanticKernel.Data;
+using Microsoft.SemanticKernel.Embeddings;
+
+namespace GettingStartedWithTextSearch;
+
+/// <summary>
+/// This example shows how to create a <see cref="ITextSearch"/> from a
+/// <see cref="IVectorStore"/>.
+/// </summary>
+public class Step4_Search_With_VectorStore(ITestOutputHelper output) : BaseTest(output)
```

Comment:
For the next iteration, it might be nice to have a vector store based function calling example as well.